pkgs/applications/science/astronomy/gildas/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl, gtk2 , pkgconfig , python27 , gfortran , python27Packages , lesstif , cfitsio , getopt , perl , groff }:
+
+stdenv.mkDerivation rec {
+  version = "feb17d";
```

Comment:
@bzizou In order to make your life (and that of everyone else) easier, you could e-mail the author of Gildas. Alternatively, you could parse the format they are using and compute the version based on their version. This way, if you ever automate the update process, there is no manual work required anymore.